modules/imgur.py

```diff
@@ -18,6 +22,97 @@ def _prefix(self, data):
             text += "%s " % data["account_url"]
         return text
 
+    @utils.hook("received.message.channel",
+                priority=EventManager.PRIORITY_LOW)
+    def channel_message(self, event):
+        if not event["channel"].get_setting("auto-imgur", False):
+            return
+
+        self._imgur(event)
+
+    def _imgur(self, event):
+        msg = event["message"]
+        reply = ""
+        match_gallery = REGEX_GALLERY.match(msg)
+        match_image = REGEX_IMAGE.match(msg)
+
+        if match_gallery:
+            reply = self._parse_gallery(match_gallery.group(1))
+
+        if match_image and not reply:
+            reply = self._parse_image(match_image.group(1))
+
+        if not reply:
+            return
+
+        event.eat()
+
+        self.events.on("send.stdout").call(target=event[
+            "channel"], module_name="Imgur", server=event["server"],
+                                           message=reply)
```

Comment:
incorrect indentation

Reply:
Is there some sort of linting program to do proper indentation. I have yet to find a suitable one?

Reply:
not that i know of - i do it manually